addons/barcodes_gs1_nomenclature/static/src/js/tests/barcode_parser_tests.js

```diff
@@ -251,12 +251,11 @@ QUnit.module('Barcode GS1 Parser', {
 
     QUnit.test('Test Alternative GS1 Separator (fnc1)', async function (assert) {
         assert.expect(6);
-        const barcodeNomenclature = new BarcodeParser({'nomenclature_id': 2});
+        const nomenclature = this.data['barcode.nomenclature'].records[1];
+        nomenclature.rules = this.data['barcode.rule'].records;
+        let barcodeNomenclature = new BarcodeParser({ nomenclature_id: 2, nomenclature });
```

Comment:
```suggestion
        let barcodeNomenclature = new BarcodeParser({ nomenclature });
```
I'm not too sure about this, but the `load()` function will do two RPC calls to get the `nomenclature` we already gave based on the `nomenclature_id`. I guess we could spare those for this test.

Reply:
Good idea, especially for QUnit tests where RPC should be mocked if we want to do things right